.babelrc.js

```diff
@@ -9,11 +9,11 @@ function preset(_, options = {}) {
         r("babel-preset-env"),
         {
           loose: true,
-          debug: !!options.debug,
+          debug: options.debug,
           modules: "commonjs",
+          useBuiltIns: true,
           targets: {
             node: PRODUCTION ? 4.0 : "current",
-            uglify: PRODUCTION ? true : false,
```

Comment:
since this isn't running the browser code this shouldn't be needed right? it effectively negates everything otherwise